.docker/aiida-core-base/s6-assets/init/aiida-daemon-start.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+verdi profile show
+if [ $? == 0 ]; then
+
+    # Supress rabbitmq version warning
+    # If it is built using RMQ version > 3.8.15 (as we did for the `aiida-core-with-services` image) which has the issue as described in
+    # https://github.com/aiidateam/aiida-core/wiki/RabbitMQ-version-to-use
+    # We explicitly set consumer_timeout to disabled in /etc/rabbitmq/rabbitmq.conf
+    verdi config set warnings.rabbitmq_version False
```

Comment:
Won't this be run every time the container is started? Guess this would have to be setup just once, shouldn't it? When the profile is set up the first time, just like you set `verdi config set warnings.development_version False`

Reply:
Yes, I think you are right. It is not proper to put it here which will execute every time the daemon start. 
But if the config is put in the `aiida-prepare.sh` as it did before. Then if the `SETUP_DEFAULT_AIIDA_PROFILE` is set to false, the config will also be skipped and users will always see two warnings if the profile is not set by default. 

I think more properly is that the `verdi config` can run independent of profile, but I think it is not the case at the moment correct? I run `verdi config set warnings.rabbitmq_version False` without profile, I got:

```
Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/site-packages/aiida/common/extendeddicts.py", line 49, in __getattr__
    return self[attr]
KeyError: 'profile'
```

Reply:
Can you try with the `--global` flag? By default `verdi config set` will apply it to the current default profile, which if that doesn't exist of course fails. But maybe setting globally will work.

Reply:
I found an issue of it https://github.com/aiidateam/aiida-core/issues/6273.
After fixing it, I can move the config set out from profile setting and `--global` will passed automatically if the profile not exist.

Reply:
The `--global` is not needed, if the profile not exist the config set will fallback to set as global. I update the PR, ready for the review.